edge-apps/cap-alerting/screenly.yml

```
@@ -0,0 +1,58 @@
+---
+syntax: manifest_v1
+id: 01KBT555AYQE42KBXJ4NPH87S0
+description: Display CAP emergency alerts on Screenly screens. Supports offline mode and uses screen tags (e.g., exit:North Lobby) to highlight the nearest exit.
+icon: https://playground.srly.io/edge-apps/cap-alerting/static/cap-icon.svg
+author: Screenly, Inc.
+ready_signal: true
+entrypoint:
+  type: file
+settings:
+  cap_feed_url:
+    type: string
+    title: CAP Feed URL
+    optional: true
+    default_value: ''
+    help_text: URL or relative path to a CAP XML feed. Leave blank to use demo mode.
+  refresh_interval:
+    type: string
+    title: Refresh Interval (minutes)
+    optional: true
+    default_value: '5'
+    help_text: Time in minutes between feed updates.
+  language:
+    type: string
+    title: Default Language
+    optional: true
+    default_value: en
+    help_text: Choose the preferred language when multiple info blocks exist (e.g., en, es, fr).
+  max_alerts:
+    type: string
+    title: Maximum Alerts
+    optional: true
+    default_value: '3'
+    help_text: Maximum number of alerts to display simultaneously.
+  audio_alert:
+    type: string
+    title: Play Audio Alerts
+    optional: true
+    default_value: 'false'
+    help_text: Play audio from CAP resources when available (true/false).
```

```suggestion
  mute_sound:
    type: string
    title: Play Audio Alerts
    optional: true
    default_value: 'false'
    help_text: Play audio from CAP resources when available (true/false). Not available for Screenly Anywhere.
```

Probably, audio alerts should be on by default. 
Also, not available on Anywhere yet.